style.css

```diff
@@ -0,0 +1,393 @@
+.menu {
+  display: flex;
+  justify-content: right;
+  padding: 20px;
+}
+
+.page {
+  display: flex;
+  flex-direction: column;
+  margin: unset;
+  overflow: auto;
+  margin-right: 24px;
+  margin-left: -24px;
+}
```

Comment:
[OPTIONAL]
- [ ] The margins you have set here is what is causing your project design to be distorted and not correctly aligned. Please look into the `px` used in line 12 & 13 and the margin value fot line 10 and see how you can fix up your project's design before merging :+1: